src/afl-fuzz-run.c

```diff
@@ -142,18 +142,82 @@ static void write_with_gap(afl_state_t *afl, void *mem, u32 len, u32 skip_at,
   s32 fd = afl->fsrv.out_fd;
   u32 tail_len = len - skip_at - skip_len;
 
+  /*
+  This memory is used to carry out the post_processing(if present) after copying
+  the testcase by removing the gaps
+  */
+  u8 mem_trimmed[skip_at + tail_len +
```

Comment:
So far, we tried to avoid the use of variable length arrays, as they are error-prone (mainly, the can be too large for the stack). I think we may want to reuse one of the alrady allocated buffers in `alf_state_t` or allocate a new one (?)

Reply:
Alternatively, the array could be fixed size to the maximum testcase size, that feels even cleaner.

Reply:
I didn't know about the allocated buffers in `afl_state_t` actually. 😅

Reply:
You can safely do  `u8 *mem_trimmed = ck_maybe_grow(BUF_PARAMS(trim), skip_at + tail_len + 1);`